requirements/test.txt

```diff
@@ -421,15 +430,15 @@ nltk==3.9.1
     # via
     #   -r requirements/base.txt
     #   unstructured
-notebook==7.3.2
+notebook==7.4.3
     # via jupyter
 notebook-shim==0.2.4
     # via
     #   jupyterlab
     #   notebook
 numpy==1.26.4
     # via
-    #   -c requirements/constraints.in
+    #   -c /Users/jiajunxu/code/unstructured-api/requirements/constraints.in
```

Comment:
pip-compile loves to add absolute paths in the comments, so you need to delete them all :D